doc/source/installation.rst

```diff
@@ -94,6 +94,32 @@ Fedora/Rawhide:
 
      $ sudo dnf install kiwi-cli
 
+
+
+Installation for SUSE Linux Enterprise Server
+---------------------------------------------
+
+{kiwi} is available and supported for SUSE Linux Enterprise Server (SLES).
```

Comment:
`SUSE Linux Enterprise Server (SLES)` -> `SUSE Linux Enterprise (SLE)`

Reply:
Fixed in commit 31b599c71.